BookEcommerce.Models/DAL/Interfaces/IProductPriceRepository.cs

```diff
@@ -9,6 +9,6 @@ namespace BookEcommerce.Models.DAL.Interfaces
 {
     public interface IProductPriceRepository : IRepository<ProductPrice>
     {
-
+        Task<ProductPrice> GetProductPricesByProductVariantId(string idProductVariant);
```

Comment:
I saw method name using ProductVariantId, but param is idProductVariant. Please make sure you guys using same naming for all. Shoube be [Object]Guid or [Object]Properties